### ISSUE TITLE
Questions about the TED on the KITTI test set

### ISSUE THREAD

**Issue by jsc122** [hailanyi/TED#24]
TED is really a great job, but I have a small problem.

That is, " **_training data_** " in " These models are not suitable to directly report results on KITTI test set, please use slightly lower score threshold and train the models on _**all or 80% training data**_ to achieve a desirable performance on KITTI test set. " refers to the KITTI training set (7481 samples) ? Or the train split (3712 samples)?

Looking forward to your reply!

**Comment by hailanyi** [comment]
All training set (7481 samples ).

**Comment by jsc122** [comment]
> All training set (7481 samples ).

Thank you for your reply.
